python/tk_framework_desktopserver/server.py

```diff
@@ -106,15 +121,18 @@ def _start_server(self):
         self.context_factory = ssl.DefaultOpenSSLContextFactory(cert_key_path,
                                                                 cert_crt_path)
 
+        # FIXME: Seems like the debugging flags are gone from the initializer at the moment.
+        # We should try to restore these.
         self.factory = WebSocketServerFactory(
-            "wss://localhost:%d" % self._port, debug=self._debug, debugCodePaths=self._debug
+            "wss://localhost:%d" % self._port # , debug=self._debug, debugCodePaths=self._debug
         )
 
         self.factory.protocol = ServerProtocol
         self.factory.host = self._host
         self.factory.user_id = self._user_id
         self.factory.notifier = self.notifier
-        self.factory.setProtocolOptions(allowHixie76=True, echoCloseCodeReason=True)
```

Comment:
hixie76 adds support for the draft implementation of websocket. that support has been removed from Twisted it seems.